tests/Project1434.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+# ########################## Copyrights and license ############################
+#                                                                              #
+# Copyright 2020 Anuj Bansal <bansalanuj1996@gmail.com>                        #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.github.io/PyGithub/v1/index.html                             #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+# ##############################################################################
+
+from . import Framework
+
+
+class Project1434(Framework.TestCase):
+    def setUp(self):
+        super().setUp()
+
+    def testDelete(self):
+        project = self.g.get_project(4102095)
+        project.delete()
```

Comment:
As above -- we should be asserting the function return type too.

Reply:
As mentioned here, https://github.com/PyGithub/PyGithub/pull/1434#discussion_r392459033, the delete method doesn't return a boolean. Not sure if it would be possible without that.

Checked other tests as well.

https://github.com/PyGithub/PyGithub/blob/37122cf2bbde825e57e78ee37171bab150802eca/tests/Reaction.py#L54-L55

https://github.com/PyGithub/PyGithub/blob/37122cf2bbde825e57e78ee37171bab150802eca/tests/GitRef.py#L69-L70

https://github.com/PyGithub/PyGithub/blob/37122cf2bbde825e57e78ee37171bab150802eca/tests/GistComment.py#L68-L69